packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -764,13 +895,36 @@ export class StatsAnalyzer extends EventsScope {
               `StatsAnalyzer:index#compareLastStatsResult --> No share frames received`,
               currentFramesReceived
             );
+            if (this.lastEmittedStartStopEvent.share.remote === EVENTS.REMOTE_MEDIA_STARTED) {
+              this.emit(
+                {
+                  file: 'statsAnalyzer',
+                  function: 'compareLastStatsResult',
+                },
+                EVENTS.NO_FRAMES_RECEIVED,
+                {
+                  mediaType: 'share',
+                }
+              );
+            }
           }
-
           if (currentFramesDecoded === previousFramesDecoded || currentFramesDecoded === 0) {
             LoggerProxy.logger.info(
               `StatsAnalyzer:index#compareLastStatsResult --> No share frames decoded`,
               currentFramesDecoded
             );
+            if (this.lastEmittedStartStopEvent.video.remote === EVENTS.REMOTE_MEDIA_STARTED) {
```

Comment:
this should be "share" not "video"

Reply:
why mistakes like that are not caught by unit tests?

Reply:
Fixed this. The UTs were failing.